modules/notification/js/notification_service.js

```diff
@@ -158,6 +175,19 @@
                     .append(notificationAction);
             }
 
+            return notification;
+        }
+
+        function notify(_text, _icon, _sticky, _color, _action, _callback, _delay)
+        {
+            var $compile = $injector.get('$compile');
+            // Use of `this` for override purpose.
+            var notification = this.buildNotification(_text, _icon, _color, _action);
```

Comment:
use `service` not this? otherwise we get this jshint error:

`Line 185, Col 32  If a strict mode function is executed using function invocation, its 'this' value will be undefined.`